includes/types/CMB2_Type_Text_Datetime_Timestamp_Timezone.php

```diff
@@ -34,6 +34,15 @@ public function render() {
 			$tz       = $datetime->getTimezone();
 			$tzstring = $tz->getName();
 			$value    = $datetime->getTimestamp();
```

Comment:
I fixed this issue of not displaying all dates and time correctly after saved by changing line 41 to:

$value = gmdate('m/d/Y h:i A', $datetime->getTimestamp());

Now all my dates show correctly and are still saved in the DB correctly: https://capture.dropbox.com/9QH9bxgAuUTPgVGt?src=ss rather than like this: https://capture.dropbox.com/Zl6aPMzCCo5rx4MY?src=ss

Reply:
Also "text_datetime_timestamp" fields have the same bug, any chance to fix also CMB2_Type_Text_Datetime_Timestamp.php 
I am trying to apply your fix to the value, but it seems not working, could you help me please?

Reply:
Can you test the changes just pushed to the develop branch to confirm the issue is fixed for your use-cases?